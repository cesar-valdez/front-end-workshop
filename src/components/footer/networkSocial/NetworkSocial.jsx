import React from 'react';
import PropTypes from 'prop-types';
import 'evil-icons/assets/evil-icons.js';

import './../footer.scss';

const NetworkSocial = props => (
	<li className="yta_Footer-socialNetwork-item">
		<a href={props.url}>
			<span data-icon={`ei-sc-${props.name}`} data-size="m"></span>
			<span className="yta_u-onlyRead">{props.name}</span>
		</a>
	</li>
)

export default NetworkSocial;

NetworkSocial.propTypes = {
	url: PropTypes.string.isRequired,
	name: PropTypes.string.isRequired,
};