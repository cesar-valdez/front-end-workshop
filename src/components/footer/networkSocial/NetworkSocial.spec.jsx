import React from 'react';
import { shallow } from 'enzyme';

import NetworkSocial from './NetworkSocial';

describe('NetworkSocial', () => {
	const networkSocial = shallow(<NetworkSocial url="www.facebok.com" name="facebook"/>);

	it('should render NetworkSocial component', () => {
		expect(networkSocial).toMatchSnapshot();
	});
});