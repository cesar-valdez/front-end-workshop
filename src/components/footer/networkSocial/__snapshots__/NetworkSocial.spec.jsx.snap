// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetworkSocial should render NetworkSocial component 1`] = `
<li
  className="yta_Footer-socialNetwork-item"
>
  <a
    href="www.facebok.com"
  >
    <span
      data-icon="ei-sc-facebook"
      data-size="m"
    />
    <span
      className="yta_u-onlyRead"
    >
      facebook
    </span>
  </a>
</li>
`;
